content/webapp/pages/exhibition-guide.tsx

```diff
@@ -86,15 +88,67 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     }
   };
 
-const ExhibitionStops = props => {
-  const { type, stops } = props;
+const Stops = ({ stops, type }) => {
+  return (
+    <ul className="plain-list no-margin no-padding">
+      {stops.map((stop, index) => {
+        const {
+          title,
+          number,
+          audioWithDescription,
+          audioWithoutDescription,
+          bsl,
+        } = stop;
+        const hasContentOfDesiredType =
+          (type === 'audio-with-descriptions' && audioWithDescription.url) ||
+          (type === 'audio-without-descriptions' &&
+            audioWithoutDescription.url) ||
+          (type === 'bsl' && bsl.embedUrl);
+        return (
+          <li key={index}>
+            <h2>
+              {number}. {title}
+            </h2>
+            {hasContentOfDesiredType ? (
+              <>
+                {type === 'audio-with-descriptions' &&
+                  audioWithDescription.url && (
+                    <AudioPlayer
+                      title={stop.title} // TODO option not to display title
```

Comment:
Do you mean giving editors an option in Prismic UI, what did you have in mind?

Reply:
No, It's just that sometimes we'll want the component to render the title and other times not, but we'll know that at the point we use it. If you look at the screenshot above, you can see the title gets displayed twice, once by the code for the 'Stop' and once by the AudioPlayer component itself. In this instance, I think we'll not want the AudioPlayer to display it.